apps/video/js/video.js

```diff
@@ -35,7 +43,6 @@ ids.forEach(function createElementRef(name) {
 
 dom.player.mozAudioChannelType = 'content';
 
-function $(id) { return document.getElementById(id); }
```

Comment:
I moved it to the shared/ file because that's where it's being used, it's not being used in the video app code.